lib/cocoapods-core/specification/root_attribute_accessors.rb

```diff
@@ -40,12 +40,17 @@ def version
           end
         end
 
-        # @return [Version] The swift_versions required to use the specification.
-        #
-        def swift_version
```

Comment:
do we need to keep this accessor around for backwards compatibility? any library that was consuming specs could potentially be reading `spec.swift_version`. The singular _setter_ is supported by the attribute DSL, but not the accessor

Reply:
good observation. I can actually keep it and delegate to the new one?

Reply:
yeah, but since the new one returns an array that might not be great either. I don't have any ideas on how to make that a smooth experience

Reply:
either:

a) return an array, but hope that libraries consuming the old `swift_version` can handle an array (probably not the best idea)
b) return `min`, `max`, `first`, or `last` as a fallback to get a singular value. also not great, but at least it's compatible

Reply:
whatever we decide to go with, that could also use a test as well

Reply:
added a test and brought it back!